sim/src/cap.rs

```diff
@@ -1,16 +1,9 @@
 use crate::{CarID, VehicleType};
 use geom::{Duration, Time};
-use map_model::{LaneID, Map, Path, PathConstraints, PathStep};
+use map_model::{LaneID, Map, Path, PathConstraints, PathRequest, PathStep};
 use serde::{Deserialize, Serialize};
 use std::collections::{BTreeMap, BTreeSet};
 
-// TODO When do we increase the counter for a zone, and when do we enforce the check? If we check
```

Comment:
I took out the old system, which bumped the counter when a car actually entered the zone. It sounds like GLT wants to have some cars reserve way in advance (for the use case of regular commuters) and some around when they start their trip.